website/views.py

```diff
@@ -135,6 +185,7 @@ def project_selection(request):
         formset = FilterFormSet(request.POST)
         if p_form.is_valid() and formset.is_valid():
             selector = ProjectSelector()
+            selector.gen_slug()
```

Comment:
can this be done automatically every time we make a new project selector? so we dont accidentally forget to call it somewhere